src/components/Text/variants.ts

```diff
@@ -57,7 +57,6 @@ export const variants: Record<TextVariant, TextProp> = {
       fontSize: '1.5rem',
       fontStyle: 'normal',
       fontWeight: '700',
-      lineHeight: '2.2rem',
```

Comment:
For this one h2- i removed the lineHeight already.

Reply:
Because there was unnecessary margin for the text-element. I wanted the margin to be 0 for all elements.
Its easier to add a margin for an element.
But if the default margin is a lot, the styles become hard to understand.

Reply:
Okay! Should the line height be removed from the other header variants too? 🤔

Reply:
Yeah, but lets do it in a separate PR

We need to check through the whole app and see if some styles are affected :S 
But it will be a huge improvement for the development experience for the future.